test/test_unit_telemetry_oob.py

```diff
@@ -104,6 +106,15 @@ def test_telemetry_oob_http_log(telemetry_setup):
     assert telemetry.size() == 0
 
 
+def test_telemetry_oob_error_code_mapping():
+    """Tests that all OCSP error codes have a corresponding Telemetry sub event type."""
+    ec_dict = snowflake.connector.errorcode.__dict__
+    for ec in ec_dict:
+        if not ec.startswith('__'):
+            if 254000 <= ec_dict[ec] < 255000:
+                assert ec_dict in OCSPTelemetryData.ERROR_CODE_MAP
```

Comment:
This is just an FYI, if you use `.items()` then you don't need to get the key and then retrieve the value with the key from the dict again.

```suggestion
    for key, value in ec_dict.items():
        if not key.startswith('__'):
            if 254000 <= value < 255000:
                assert key in OCSPTelemetryData.ERROR_CODE_MAP
```

Although I'm guessing what you wanted to to do with `ec_dict` in the last line, make sure that you wanted to check whether the whole dictionary was in the other dictionary.